src/rt/msvc.c

```diff
@@ -6,10 +6,25 @@
 * License: Distributed under the
 *      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
 *    (See accompanying file LICENSE)
-* Source:    $(DRUNTIMESRC rt/_stdio_msvc.d)
+* Source:    $(DRUNTIMESRC rt/_msvc.c)
```

Comment:
No documentation is generated from the rt package, so it's unlikely to make a difference. It's used everywhere so better keep it.